application/app/views/dataverse/datasets/_dataset_info.html.erb

```diff
@@ -1,19 +1,50 @@
-<div class="card dataset-header">
-  <div class="card-header"><%= t('dataverse.datasets.dataset_info.header_dataset_text') %></div>
-  <ul class="list-group list-group-flush">
-    <li class="list-group-item">
-       <strong><%= dataset.metadata_field('title').to_s %></strong>
-    </li>
-    <li class="list-group-item">
-       <%= dataset.data.dataset_persistent_id %>
-    </li>
-  </ul>
+<div class="card mb-3">
+  <div class="card-header">
+    <div class="d-flex justify-content-between align-items-center gap-3 flex-wrap">
+      <div class="flex-grow-1 d-flex flex-column">
+        <h2 class="mb-0 text-nowrap h5"><%= t('.header_dataset_text') %> <%= dataset.data.dataset_persistent_id %></h2>
+        <span class="text-muted" style="font-size: 0.8rem;">
+          <strong><%= dataset.metadata_field('title').to_s %></strong>
+        </span>
+      </div>
+    </div>
+    <div>
```

Comment:
Looking at the page in browser, I would add <div class="mt-2">
Same with the Collections page

Reply:
fixed in both views